src/components/Header.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import heroImage from 'assets/heroImage.jpg';
+import heroLinkedin from 'assets/linkedin-icon.svg';
+import heroGithub from 'assets/github-icon.svg';
+/* import heroStackOverflow from 'assets/stackoverflow-icon.svg'; */
+import profileImage from 'assets/profileImage-4.jpg';
+import { GITHUB_URL, LINKEDIN_URL /* , STACKOVERFLOW_URL */ } from 'utils/Urls';
+
+export const HeaderContainer = styled.header`
+  background-image: url(${heroImage});
+  width: 100%;
+  height: 450px;
+  background-position: top center;
+  background-size: cover;
+  position: relative;
+`;
+
+export const HeaderIcons = styled.div`
+  position: absolute;
+  right: 10%;
+  top: 8%;
+  
+  img{
+    height: 50px;
+    margin-right: 10px;
+    }
+
+  img:hover{
+    filter: opacity(0.5);
+  }
+
+@media (min-width: 668px) and (max-width: 1023px){
+    img{
+        height: 65px
+    }
+}
+
```

Comment:
Awesome looking header, the background was a nice touch, And i love that you added it in the introduction section also with a lighter shade

and good job on the media querys :)